docs/DEVGUIDE.md

```diff
@@ -174,137 +178,248 @@ The images are tagged with the current Git commit SHA:
+planUpdatable: false
+plans:
+- name: default
+  osbFree: true
+  osbGuid: 86064792-7ea2-467b-af93-ac9694d96d52
+```
+
+### Provisioning a new Instance
+
+Let's provision a new instance of the `user-provided-service`.  To do this, we
+create a new [`Instance`](../contrib/examples/walkthrough/ups-instance.yaml) to
+indicate that we want to provision a new instance of that service:
+
+    kubectl create -f contrib/examples/walkthrough/ups-instance.yaml
+    instance "ups-instance" created
+
+We can check the status of the `Instance` using `kubectl get`:
+
+    kubectl get instances -n test-ns ups-instance -o yaml
+
+We should see something like:
+
+```yaml
+$ k get instances ups-instance -n test-ns -o yaml                                                                                                 
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Instance
+metadata:
+  creationTimestamp: 2017-03-03T04:26:08Z
+  name: ups-instance
+  namespace: test-ns
+  resourceVersion: "9"
+  selfLink: /apis/servicecatalog.k8s.io/v1alpha1/namespaces/test-ns/instances/ups-instance
+  uid: 8654e626-ffc9-11e6-b111-0242ac110005
+spec:
+  osbGuid: 34c984e1-4626-4574-8a95-9e500d0d48d3
+  planName: default
+  serviceClassName: user-provided-service
+status:
+  conditions:
+  - message: The instance was provisioned successfully
+    reason: ProvisionedSuccessfully
+    status: "True"
+    type: Ready
+```
+
+### Bind to the Instance
+
+Now that our `Instance` has been created, let's bind to it.  To do this, we
+create a new [`Binding`](../contrib/examples/walkthrough/ups-binding.yaml).
+
+    kubectl create -f contrib/examples/walkthrough/ups-binding.yaml
+    binding "ups-binding" created
+
+We can check the status of the `Instance` using `kubectl get`:
+
+    kubectl get bindings -n test-ns ups-binding -o yaml
+
+We should see something like:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Binding
+metadata:
+  creationTimestamp: 2017-03-07T01:44:36Z
+  finalizers:
+  - kubernetes
+  name: ups-binding
+  namespace: test-ns
+  resourceVersion: "29"
+  selfLink: /apis/servicecatalog.k8s.io/v1alpha1/namespaces/test-ns/bindings/ups-binding
+  uid: 9eb2cdce-02d7-11e7-8edb-0242ac110005
+spec:
+  instanceRef:
+    name: ups-instance
+  osbGuid: b041db94-a5a0-41a2-87ae-1025ba760918
+  secretName: my-secret
+status:
+  conditions:
+  - message: Injected bind result
+    reason: InjectedBindResult
+    status: "True"
+    type: Ready
+```
+
+Notice that the status has a ready condition set.  This means our binding is
+ready to use.  If we look at the secrets in our `test-ns` namespace in
+kubernetes, we should see:
+
+    kubectl get secrets -n test-ns
+    NAME                  TYPE                                  DATA      AGE
+    default-token-3k61z   kubernetes.io/service-account-token   3         29m
+    my-secret             Opaque                                2         1m
+
+Notice that a secret named `my-secret` has been created in our namespace.
```

Comment:
In a follow-up PR we really should show using the secret in an application just to complete the picture.